test/mixin.js

```diff
@@ -16,18 +18,19 @@ var i18n = require( '..' ),
 
 describe( 'mixin()', function() {
 	useFakeDom();
+	setupEnzymeAdapter();
```

Comment:
I've removed this and tests continue to pass 🤷‍♂️ 

Fine to leave it in. Ideally I believe this would go in some test setup before any tests are run, but this works for now.

Reply:
Right, I've removed it. I still have to figure out why it actually does.